app/resources/assets/sass/_suggestion_detail.scss

```diff
@@ -0,0 +1,28 @@
+@import "_placeholders.scss";
+
+.suggestion-detail{
```

Comment:
Add spaces immediately before the { to be more consistent with the current scss files.  

I marked in red the space from a file that existed before you made any changes:
![image](https://user-images.githubusercontent.com/11838191/43258078-8d79e382-909f-11e8-93dd-dcd2708f0d0c.png)